src/pages/components/menu-buttons/usage.mdx

```diff
@@ -420,8 +423,10 @@ cases where using a menu button is recommended:
 - Actions related to a common label
 - Distinct actions applied to the same object or page
 
-Menu button and combo button come in three different button types: primary,
-tertiary, and ghost.
+Menu button comes in three different button types:
```

Comment:
I think it's redundant to provide links for three button types taking users to the same page. I don't think we use this pattern anywhere either. So maybe:

> For detailed visuals, see them in the [Style tab](/components/menu-buttons/style/#menu-button-color).